Gruntfile.coffee

```diff
@@ -1324,7 +1337,27 @@ module.exports = (grunt) ->
 					base: "<%= coreDist %>"
 					message: ((
 						if process.env.TRAVIS_TAG
-							"Variant files for the " + process.env.TRAVIS_TAG + " maintenance release"
```

Comment:
That is the message that is used for everything (it was the text in the release instructions that inspired most of this). We can change it but we will have to change in multiple places.